src/translations/zh-hans.global.json

```diff
@@ -817,7 +817,7 @@
         "pageTitle": {
             "loading": "正在加载...",
             "notFound": "账户 ${accountId} 未找到",
-            "default": "账户：${accountId}"
```

Comment:
@picturepan2 can you please confirm that this translation still makes sense without an account ID

Reply:
@picturepan2 I'm gonna include this temp fix in the current [release](https://github.com/near/near-wallet/pull/1390/) since it's better than showing the undefined variable on the UI.

![Screen Shot 2021-02-03 at 1 05 16 PM](https://user-images.githubusercontent.com/24921205/106809381-7736d180-6620-11eb-87e3-8a6b2139b2b4.png)

Reply:
Yes, it means "Account". It is good.

Reply:
@Patrick1904 Yes, it means "Account". It looks still good without account ID.